### DISCUSSION TITLE
how to remove duplicate in history

### DISCUSSION THREAD

**Discussion by WingDust** [nushell/nushell#9682] [Q&A]
i want to remove duplicate in history command , how to deal with?

**Comment by fdncred** [comment]
There is no built in mechanism to deal with it. If you search issues and discussions you'll find that some people use a sql query if you're using the sqlite history format.

**Comment by healBvdb** [comment]
here is the command that I am using to remove duplicate line from history.txt in nushhell
in your nushell config directory depending if you are on linux or windows (~/AppData/Roaming/nushell or .config/nushell)
`history | uniq-by command | get command | save -f history.txt`
